packages/components/stories/icon/icons.js

```diff
@@ -1078,4 +1078,4 @@ export const icons =
         'IcWalletUsdCoinDark',
         'IcWalletUsdCoinLight',
     ],
-}
+}
```

Comment:
This is auto generated file. isnt?
is this change really needed  

Reply:
Will replace the content with master